sim/druid/talents.go

```diff
@@ -43,6 +48,10 @@ func (druid *Druid) ApplyTalents() {
 		druid.PseudoStats.ThreatMultiplier *= 1 - 0.04*float64(druid.Talents.Subtlety)
 	}
 
+	if druid.Talents.Naturalist > 0 {
```

Comment:
Don't need the if-statement here since this will be a no-op with 0 points.

Reply:
Why is the if-statement used in the preceding lines that apply the Intensity and Subtlety talents then?

Reply:
Those can be removed too, probably just sloppy copy+paste. The if-statement does matter in some cases, for example adding a stat dependency. But PseudoStats are just raw float64 fields so as long as the equation is a no-op with 0 points then its fine.

Reply:
Sounds good, I went ahead and removed them all.